app/src/main/java/org/jellyfin/androidtv/ui/browsing/DisplayPreferencesScreen.kt

```diff
@@ -46,6 +46,19 @@ class DisplayPreferencesScreen : OptionsFragment() {
 					default { "0" }
 				}
 			}
+			list {
+				setTitle(R.string.grid_direction)
+				entries = mapOf(
+					"0" to requireContext().getString(R.string.grid_direction_vertical),
+					"1" to requireContext().getString(R.string.grid_direction_horizontal),
+				)
+
+				bind {
+					get { displayPreferences.getCustomPrefs().get("GridDirection") ?: "0" }
+					set { displayPreferences.getCustomPrefs().set("GridDirection", it) }
+					default { "0" }
+				}
```

Comment:
I'm guessing you changed from an enum to integers because of this binding part. I'd rather use enums in the code because it's type safe. You can then store the enum names in the displaypreferences.